css/style.css

```diff
@@ -72,6 +72,10 @@ html {
     opacity: 0.75;
 }
 
+.mouse__preview.blocked {

```

Comment:
wouldn't `blocked` be a modifier on the preview? eg. `.mouse__preview--blocked`

not 100% sure how BEM works for this kind of thing.

Reply:
I don't know BEM, is this project strict about BEM usage?